docs/source/installation.md

```diff
@@ -67,31 +67,15 @@ pip install datasets[audio]
 
 <Tip warning={true}>
 
-On Linux, non-Python dependency on `libsndfile` package must be installed manually, using your distribution package manager, for example:
+To decode mp3 files you should have the `libsndfile` system library of at least version 1.1.0. Normally, it's bundled into the python [`soundfile`](https://github.com/bastibe/python-soundfile) package which is installed as an extra audio dependency of 🤗 Datasets.
+On Linux, the required version of `libsndfile` is bundled into the `soundfile` wheel since version 0.12.0 of `soundfile`. You can check which version of `libsndfile` is used by `soundfile` running the following code:
```

Comment:
```suggestion
On Linux, the required version of `libsndfile` is bundled into the `soundfile` wheel since version 0.12.0 of `soundfile`. You can run the following command to check which version of `libsndfile` is used by `soundfile`:
```